### DISCUSSION TITLE
Lines Problem

### DISCUSSION THREAD

**Discussion by Chauya** [slyh/Cities2-TrafficLightsEnhancement#94] [General]
Hello ! 
I have a problem with the mod
I've done this interchange here : 
![image](https://github.com/slyh/Cities2-TrafficLightsEnhancement/assets/105445800/4a7eb701-73c2-47ce-bc28-80a4e9d4d4e9)
i try to remove the left turn from the road comming from the bottom, but when i do this with the mod, this is what i get :
![image](https://github.com/slyh/Cities2-TrafficLightsEnhancement/assets/105445800/1bea646c-d5b3-42a4-87b8-dee515e61254)
Idk why the 2 lanes are merging, dont know how to make them go straight...
Is it a bug or it's just me ?

**Comment by slyh** [comment]
I believe the TMPE 2 by krzychu124 is close to being released. Once ready, the Lane Direction Tool will be retired.

**Reply by Chauya** [reply]
okay thx
